src/Arcus.Security.Providers.AzureKeyVault/Extensions/SecretStoreBuilderExtensions.cs

```diff
@@ -109,9 +111,10 @@ public static class SecretStoreBuilderExtensions
             string tenantId,
             string clientId,
             X509Certificate2 certificate,
-            bool allowCaching = false,
-            Func<string, string> mutateSecretName = null,
-            Action<KeyVaultOptions> configureOptions = null)
+            Action<KeyVaultOptions> configureOptions,
```

Comment:
Can we move this to the end?

Reply:
We could but we would mixing up specific with non-specific parameters. Not sure if that's a good idea?

Reply:
Can you elaborate on it please, not sure if I'm following!

Reply:
The 'specific' Azure Key Vault parameters (`TokenCredential`, `clientId`, `clientSecret`, `KeyVaultOptions`...) are parameters I would want first in the signature, before the 'general' parameters (`mutateSecretName`, `name`, (future) `optional`...).

Make sense?

Reply:
Fine by me!